modules/net-address/outputs.tf

```diff
@@ -18,54 +18,38 @@ output "external_addresses" {
   description = "Allocated external addresses."
   value = {
     for address in google_compute_address.external :
-    address.name => {
-      address   = address.address
-      self_link = address.self_link
-    }
+    address.name => address
```

Comment:
self links are used in a few places, can we add a second set of outputs for ids?

Reply:
god I'm dumb... :)